src/test/java/roomescape/unit/service/ReservationServiceTest.java

```diff
@@ -0,0 +1,157 @@
+package roomescape.unit.service;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.anyLong;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static roomescape.common.Constant.FIXED_CLOCK;
+import static roomescape.unit.fixture.ReservationDateFixture.예약날짜_오늘;
+
+import java.time.Clock;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import roomescape.controller.rest.request.ReservationCreateRequest;
+import roomescape.controller.rest.response.ReservationResponse;
+import roomescape.domain.reservation.Reservation;
+import roomescape.repository.ReservationRepository;
+import roomescape.service.ReservationService;
+import roomescape.domain.theme.Theme;
+import roomescape.domain.theme.ThemeDescription;
+import roomescape.domain.theme.ThemeName;
+import roomescape.domain.theme.ThemeThumbnail;
+import roomescape.repository.ThemeRepository;
+import roomescape.domain.time.ReservationTime;
+import roomescape.repository.ReservationTimeRepository;
+
+public class ReservationServiceTest {
+
+    private ReservationRepository reservationRepository;
+    private ReservationTimeRepository reservationTimeRepository;
+    private ThemeRepository themeRepository;
+    private Clock clock;
+    private ReservationService service;
+
+    private final LocalTime time = LocalTime.of(10, 0);
+
+    @BeforeEach
+    void setUp() {
+        reservationRepository = mock(ReservationRepository.class);
+        reservationTimeRepository = mock(ReservationTimeRepository.class);
+        themeRepository = mock(ThemeRepository.class);
+        clock = FIXED_CLOCK;
+        service = new ReservationService(reservationRepository, reservationTimeRepository, themeRepository, clock);
+    }
```

Comment:
```suggestion
    private final ReservationRepository reservationRepository = mock(ReservationRepository.class);
    private final ReservationTimeRepository reservationTimeRepository = mock(ReservationTimeRepository.class);
    private final ThemeRepository themeRepository = mock(ThemeRepository.class);
    private final ReservationService service = new ReservationService(reservationRepository, reservationTimeRepository, themeRepository, FIXED_CLOCK);

    private final LocalTime time = LocalTime.of(10, 0);
```

https://github.com/woowacourse/spring-roomescape-member/pull/182#discussion_r2070340273